### ISSUE TITLE
Emscripten Touch events in WebGL port of Panda

### ISSUE THREAD

**Issue by Max-Rodriguez** [panda3d/panda3d#1509]
## Description
Implement interacting with the Panda window and graphical interface through Emscripten touch events for mobile devices running the WebGL port of Panda.

## Use Case
The WebGL port of Panda is incredibly useful for making applications available on mobile devices through the web. The only way for mobile devices to interact with the Panda window is through the touch screen, which does not send the same events as a mouse does on the desktop, which makes Panda applications currently unusable on mobile devices. The Emscripten API should allow Panda to pick up these mobile touch events. There are currently no workarounds to interacting with the window on mobile.

**Comment by Max-Rodriguez** [comment]
I'm aware the webgl-port branch is experimental and in development. Thought I'd still create this issue to bring attention to this feature.

**Comment by rdb** [comment]
Yes, and this should probably be done on top of the work that's been done in the multitouch branch, though it hasn't been touched in a while.

**Comment by Kupoman** [comment]
+1
This would be very nice to have.
